docs-source/content/userguide/introduction/introduction.md

```diff
@@ -48,6 +50,22 @@ In accordance with these policies, the operator and WebLogic Server are supporte
 Infrastructure using *Oracle Container Engine for Kubernetes*, or in a cluster running *Oracle Linux
 Container Services for use with Kubernetes* on OCI Compute, and on "authorized cloud environments".
 
+### Microsoft Azure Kubernetes Service
+
+[Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/) is a hosted Kubernetes environment.  The WebLogic Kubernetes
+operator, Oracle WebLogic Sever 12c and Oracle Fusion Middleware Infrastructure 12c are fully supported and certified on Azure Kubernetes Service (as per the documents
```

Comment:
"operator" -> "Operator"

Reply:
i thought we used lower case? @rosemarymarano